--- adams-bridge/src/mldsa_top/rtl/mldsa_seq_sec.sv	2025-05-27 17:33:54.252318448 +0000
+++ rtl/mldsa_seq_sec.sv	2025-05-27 17:46:53.140489725 +0000
@@ -30,6 +30,8 @@
   output mldsa_seq_instr_t data_o
   );
 
+    //  MJOS: debug address decoder
+    mldsa_seq_decode_sec dec_sec(clk, en_i, addr_i);
 
 `ifdef RV_FPGA_OPTIMIZE
     (*rom_style = "block" *) mldsa_seq_instr_t data_o_rom;
@@ -199,4 +201,4 @@
     end
 end
 
-endmodule
\ No newline at end of file
+endmodule
